clang-tools-extra/clangd/tool/ClangdMain.cpp

```diff
@@ -242,13 +242,13 @@ opt<std::string> FallbackStyle{
     init(clang::format::DefaultFallbackStyle),
 };
 
-opt<int> EnableFunctionArgSnippets{
+opt<std::string> EnableFunctionArgSnippets{
```

Comment:
We could also consider the following option, but up to you.
```
enum FunctionArgSnippetsFlag { True, False, None, ... };
opt<FunctionArgSnippetsFlag> EnableFunctionArgSnippets{...
  values(
    clEnumValN(FunctionArgSnippetsFlag::True, "true", ...),
    clEnumValN(FunctionArgSnippetsFlag::True, "True", ...),
    ...
  ),
};
  
```

Reply:
Thanks for the suggestion. I tried this briefly, but I think it makes the `--help` output too noisy because it prints a separate line for each spelling of `"true"`, `"True"`, etc.